src/UuidEncoding/composer.json

```diff
@@ -15,10 +15,5 @@
         "psr-4": {
             "EventSauce\\UuidEncoding\\": "./"
         }
-    },
-    "extra": {
```

Comment:
Could we update these branch alias extra configs to `"dev-main": "1.x-dev"` instead of removing them?